### ISSUE TITLE
Could not create task ':react-native-job-queue:spotlessJava

### ISSUE THREAD

**Issue by Dhanyalalithamony** [SimonErm/react-native-job-queue#102]
Could not create task ':react-native-job-queue:spotlessJava'.
   > java.nio.file.NoSuchFileException: /Users/suryabhadri/active-oversight-mobile/node_modules/react-native-job-queue/android/licenseHeader.txt

react-native-job-queue version - 0.5.1

**Comment by KartikDevarde2601** [comment]
i am also facing same error do you figure out  yet

**Comment by KartikDevarde2601** [comment]
hii Dhanyala ,
error is due to the jCenter. it has been sunsetted  so gradle can not resolve dependencies for the Spotless plugin  getting  502 Bad Gateway error from the JCenter repository 
```
repositories {
        google()
       // jcenter() comment of jcenter 
    }
```

**Comment by SimonErm** [comment]
Fixed in the next version by #104

**Comment by natemartins** [comment]
This issue persists. I have version `0.5.2` installed.
